framework/test/test_bug_mining.sh

```diff
@@ -358,8 +359,9 @@ test_promote_to_db() {
     local project_build_xml="projects/$project_id/$project_id.build.xml"
     _check_output "$HERE/../$project_build_xml" "$work_dir/framework/$project_build_xml"
 
-    [ -s "$HERE/../projects/$project_id/commit-db" ] || die "Commit-db does not exit or it is empty"
-    [ -s "$HERE/../projects/$project_id/dir-layout.csv" ] || die "dir-layout.csv does not exit or it is empty"
+    [ -s "$HERE/../projects/$project_id/$BUGS_CSV_ACTIVE" ] || die "active-bugs csv does not exist or it is empty"
+    [ -s "$HERE/../projects/$project_id/$BUGS_CSV_DEPRECATED" ] || die "deprecated-bugs csv does not exist or it is missing the header"
+    [ -s "$HERE/../projects/$project_id/dir-layout.csv" ] || die "dir-layout.csv does not exist or it is empty"
```

Comment:
Would be good to have a constant for this file as well -- maybe in a dedicated PR that cleans up the metadata (see 
also #309).

Reply:
For `dir_layout.csv`? I can add one if you'd like.

Reply:
Added constant for `dir_layout.csv` in latest commit.